stubs/date.stub

```diff
@@ -9,3 +9,27 @@ class DateInterval
 	public $days;
 
 }
+
+/**
+ * @template TEnd of ?DateTimeInterface
+ * @template TRecurrences of ?int
+ */
+class DatePeriod
```

Comment:
One more thing - `DatePeriod` needs to be added here in the config: https://github.com/phpstan/phpstan-src/blob/d92d6162dd865735609480eea89ca1219bf56ce9/conf/config.neon#L25

Reply:
Simiarly to what was done here: https://github.com/phpstan/phpstan-doctrine/commit/8061e5fdec01de856fd660aea2a9c9f447368716